src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -231,16 +231,30 @@ private void Write(BuildMessageEventArgs e)
                 Write((CriticalBuildMessageEventArgs)e);
                 return;
             }
-            else if (e is TaskCommandLineEventArgs)
+            if (e is TaskCommandLineEventArgs)
```

Comment:
Nit: it'd be nice to have an empty line to separate the two if statements visually.